test/chain-adapters/Optimism_Adapter.ts

```diff
@@ -0,0 +1,86 @@
+import { sampleL2Gas, amountToLp, mockTreeRoot, refundProposalLiveness, bondAmount } from "./../constants";
```

Comment:
@nicholaspai take a look at this test file. it uses my newly created `createMock` function that returns a `FakeContract`